Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Console = System.Console;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            Descriptor();
+            Descriptor();
+        }
+
+        public static void Descriptor()
```

Comment:
Is this really a good name for a method? I wouldn't know what it does without looking at the contents, please pick a better name.

Reply:
Changed the method name to Interview(), it's as good of a name I can come up with.